CodeAnalyzerTool/CodeAnalyzerService.Backend/DTOs/Mappers/AnalysisMapper.cs

```diff
@@ -3,9 +3,9 @@
 
 namespace CodeAnalyzerService.Backend.Dtos.Mappers;
 
-public class AnalysisMapper
+public static class AnalysisMapper
 {
-    public static AnalysisResponse MapToDto(Analysis analysis)
+    public static AnalysisResponse MapToDto(this Analysis analysis)
```

Comment:
Waarom this syntax? Is preference en ik had voor de andere gekozen omdat het dan duidelijker is welke mapper er gebruikt wordt